catchup/ledgerFetcher.go

```diff
@@ -39,12 +40,8 @@ var errNoLedgerForRound = errors.New("No ledger available for given round")
 const (
 	// maxCatchpointFileChunkSize is a rough estimate for the worst-case scenario we're going to have of all the accounts data per a single catchpoint file chunk.
 	maxCatchpointFileChunkSize = ledger.BalancesPerCatchpointFileChunk * basics.MaxEncodedAccountDataSize
-	// maxCatchpointFileDownloadDuration is the maximum time we would wait for download an entire catchpoint file
-	maxCatchpointFileDownloadDuration = 45 * time.Minute
-	// expectedWorstDownloadSpeedBytesPerSecond defines the worst case-scenario download speed we expect to get while downloading a catchpoint file
-	expectedWorstDownloadSpeedBytesPerSecond = 20 * 1024
-	// maxCatchpointFileChunkDownloadDuration is the maximum amount of time we would wait to download a single chunk off a catchpoint file
-	maxCatchpointFileChunkDownloadDuration = 2*time.Minute + maxCatchpointFileChunkSize*time.Second/expectedWorstDownloadSpeedBytesPerSecond
+	// defaultMinCatchpointFileDownloadRate defines the worst case-scenario download speed we expect to get while downloading a catchpoint file
+	defaultMinCatchpointFileDownloadRate = 20 * 1024
```

Comment:
To be consistent with the server, and eliminate the ambiguity of the units, I prefer to call this defaultMinCatchpointFileDownloadBytesPerSecond.